src/sql/foglamp_ddl.sql

```diff
@@ -877,10 +878,16 @@ ALTER TABLE foglamp.schedules OWNER to foglamp;
 CREATE TABLE foglamp.tasks (
   id           uuid                        UNIQUE,                 -- Unique uuid, PK
   process_name character varying(20)       NOT NULL,               -- Name of the task
-  state        smallint                    NOT NULL,               -- State of the task: 1-Running, 2-Complete, 3-Cancelled
+  state        smallint                    NOT NULL,               -- State of the task: 1-Running, 2-Complete, 3-Cancelled Terris:
+                                                                   -- Terris: should there be a "didn't start" state
+                                                                   -- or is this only for tasks that started?
   start_time   timestamp(6) with time zone NOT NULL DEFAULT now(), -- The date and time the task started
   end_time     timestamp(6) with time zone,                        -- The date and time the task ended
   reason       character varying(20),                              -- The reason why the task ended
+                                                                   -- Terris: Please enumerate reason codes here
```

Comment:
I would say character 20 is too small. This was meant to be the place that a task could write a message that was task specific, not a code as such. Probably not well explained in the document.